byron/semantics/executable-spec/src/Control/State/Transition/Generator.hs

```diff
@@ -300,6 +303,29 @@ genTraceOfLength aTraceLength profile env st0 aSigGen =
         (mkTrace env st0 (zip rootStates rootSignals))
         (fmap (fmap (closure @s env st0)) children)
 
+-- | Generate an invalid trace
+--
+invalidTrace
+  :: forall s
+   . HasTrace s
+  => Word64
+  -- ^ Maximum length of the generated traces.
+  -> [(Int, PredicateFailure s)]
+  -- ^ Trace failure profile to be used to get an invalid signal.
```

Comment:
And here, we only use part of the profile.

Reply:
Sorry, I don't get this comment :grimacing: